samples/zigbee/network_coordinator/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Use software cryptography on nRF5340
+CONFIG_CRYPTO=n
+CONFIG_CRYPTO_NRF_ECB=n
+CONFIG_ZIGBEE_USE_SOFTWARE_AES=y
```

Comment:
`prj.conf` will still include `CONFIG_CRYPTO_INIT_PRIORITY=80`, still need to check the warnings